exe.c

```diff
@@ -468,13 +475,13 @@ void cgc_add_enclosing_loop(int loop_fs, int break_lbl, ast continue_lbl) {
   cgc_locals = binding;
 }
 
-void cgc_add_enclosing_switch(int loop_fs, int break_lbl) {
+void cgc_add_enclosing_switch(int loop_fs, int break_lbl, int next_case_lbl) {
   int binding = alloc_obj(5);
   heap[binding+0] = cgc_locals;
   heap[binding+1] = BINDING_SWITCH;
   heap[binding+2] = loop_fs;
   heap[binding+3] = break_lbl;
-  heap[binding+4] = 0;
+  heap[binding+4] = next_case_lbl;
```

Comment:
Why was this 0 before?

Reply:
I had the same question. I tried removing it and shrinking the object and it still worked so I think it was left by accident in a previous PR.